radicle-cli/src/commands/merge.rs

```diff
@@ -95,35 +102,31 @@ impl Args for Options {
                     interactive = true;
                 }
                 Long("revision") | Short('r') => {
-                    let value = parser.value()?;
-                    let id =
-                        RevisionIx::from_str(value.to_str().unwrap_or_default()).map_err(|_| {
-                            anyhow!("invalid revision number `{}`", value.to_string_lossy())
-                        })?;
-                    revision = Some(id);
+                    revision = true;
                 }
                 Value(val) => {
-                    let val = val
-                        .to_str()
-                        .ok_or_else(|| anyhow!("patch id specified is not UTF-8"))?;
-
-                    id = Some(
-                        PatchId::from_str(val)
-                            .map_err(|_| anyhow!("invalid patch id '{}'", val))?,
-                    );
+                    str_id = Some(val);
                 }
                 _ => return Err(anyhow::anyhow!(arg.unexpected())),
             }
         }
 
-        Ok((
-            Options {
-                id: id.ok_or_else(|| anyhow!("a patch id to merge must be provided"))?,
-                interactive,
-                revision,
-            },
-            vec![],
-        ))
+        let str_id =
+            str_id.ok_or_else(|| anyhow!("a patch or revision id to merge must be provided"))?;
```

Comment:
Now that I think about it more, I think we should only support revision-ids, since the patch id is the id of the first revision!

Reply:
Indeed, Ill update the changes to account for it